library_generation/utils/utilities.sh

```diff
@@ -198,13 +229,15 @@ download_protoc() {
 download_grpc_plugin() {
   local grpc_version=$1
   local os_architecture=$2
-  if [ ! -f "protoc-gen-grpc-java-${grpc_version}-${os_architecture}.exe" ]; then
+  grpc_filename="protoc-gen-grpc-java-${grpc_version}-${os_architecture}.exe"
+  if [ ! -f "${grpc_filename}" ]; then
     # download protoc-gen-grpc-java plugin from Google maven central mirror.
     download_from \
-    "https://maven-central.storage-download.googleapis.com/maven2/io/grpc/protoc-gen-grpc-java/${grpc_version}/protoc-gen-grpc-java-${grpc_version}-${os_architecture}.exe" \
-    "protoc-gen-grpc-java-${grpc_version}-${os_architecture}.exe"
-    chmod +x "protoc-gen-grpc-java-${grpc_version}-${os_architecture}.exe"
+    "https://maven-central.storage-download.googleapis.com/maven2/io/grpc/protoc-gen-grpc-java/${grpc_version}/${grpc_filename}" \
+    "${grpc_filename}"
+    chmod +x "${grpc_filename}"
   fi
+  echo "$(pwd)/${grpc_filename}"
```

Comment:
Is this a debug log?

Reply:
No, it's the return statement of the function. Many functions in `utilities.sh` are made this way. The way we use is `variable=$(my_function "arg1" "arg2")`. The only possible return values for `return` are integers from 0 to 255 (and consumed via `$?`) so it's not useful.